vega_sim/scenario/fuzzed_markets/scenario.py

```diff
@@ -540,6 +541,15 @@ def configure_agents(
             for _, agent_list in market_agents.items():
                 self.agents.extend(agent_list)
 
+        self.agents.append(
+            FuzzyRandomTransactionAgent(
+                wallet_name="FUZZY_RANDOM_TX",
+                key_name="RANDOM_TX_MAKER",
+                tag="TX_RND",
+                asset_name=asset_name,
+            )
+        )
+
```

Comment:
Should this be indented a level with a unique tag per market/asset? Atm in this version of the fuzzing we have a unique asset per market.